source/docs/software/advanced-controls/state-space/state-space-glossary.rst

```diff
@@ -0,0 +1,76 @@
+State-Space Glossary
+====================
+
+.. glossary::
+
+    Dynamics
+        A branch of physics concerned with the motion of bodies under the action of forces. In modern control, systems evolve according to their dynamics.
+
+    Control Effort
+        A term describing how much force, pressure, etc. an actuator is exerting
+
+    Controller
+        Applies an :term:`input` to a :term:`plant` to drive the difference between a :term:`reference` and :term:`output`, or :term:`error`, to zero.
+
+    Control Law
+        A mathematical formula that generates :term:`inputs <input>` to drive a :term:`system` to a desired :term:`state`.
+
+    Error
+        :term:`Reference` minus an output or state.
+
+    Gain
+        A proportional value that relates the magnitude of an input signal to the magnitude of an output signal. In the signal-dimensional case, gain can be thought of as the proportional term of a PID controller. A gain greater than one would amplify an input signal, while a gain less than one would dampen an input signal. A gain less than one would negate the input.
+
+    Hidden State
+        A :term:`state` that cannot be directly measured, but whose :term:`Dynamics` can be related to other states.
+
+    Input
```

Comment:
Some of these terms are duplicates of terms in `docs/software/advanced-controls/introduction/controls-glossary.rst` (Same terms; same definitions; that file is lowercase)

Duplicate definitions should be removed. Is there a standard for lowercase/uppercase terms?

Reply:
@Daltz333 might be able to comment on the standard? If they are duplicates, lets standardize on one capitalization or the other and remove the duplicates.